test/e2e/framework/consts.go

```diff
@@ -22,7 +22,7 @@ const (
 
 	// MCSLBPollTimeout defines the time to wait a MCS to be assigned with LoadBalancer IP address.
 	// As MCS depending on handling service related to cloud provider, more time is required.
-	MCSLBPollTimeout = 240 * time.Second
+	MCSLBPollTimeout = 360 * time.Second
```

Comment:
why relax this timeout? We didn't do anything on MCS

Reply:
Does cloud provider has an SLO for LB creation?

Reply:
nit: 360 is like 6min,  usually it's 1,5,10, why pick 6?

Reply:
There are some flakiness in the mcs tests, it took more time than before for the ip to be ready in the service.

I tried 5mins, and still the tests failed.

Reply:
Cloud provider uses a timeout of 5 minutes for their e2e tests: https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/tests/e2e/utils/service_utils.go#L38 And there were e2e flakiness from time to time. We can try longer timeout.

Reply:
changed to 10mins, since it's out of our control.